### PR TITLE
Created a json file describing beasts and the effecs of their parts.

### PR DESCRIPTION
**Pull request by jlevertov**
for #2

### CODE REVIEW

File: server/src/database/BeastsDatabase.json [hunk 1]
```diff
@@ -0,0 +1,330 @@
...
+      "Stats": {
+        "MS": "50"
+      },
+      "Active": {
+        "Name": "Gallop",
+        "Effects": [
+          {
+            "Type": "Jump 5 meters every step",
+            "Good": {
+              "Type": "Jump 10 meters every step and over walls"
```
Comment (golanir) [inline server/src/database/BeastsDatabase.json]: like no clip?

Comment (jlevertov) [reply server/src/database/BeastsDatabase.json]: ?

File: server/src/database/BeastsDatabase.json [hunk 2]
```diff
@@ -0,0 +1,330 @@
...
+      "Name": "Hoofs",
+      "Description": "Attach Elwyn's magical hoofs to your feet",
+      "Stats": {
+        "MS": "50"
+      },
+      "Active": {
+        "Name": "Gallop",
+        "Effects": [
+          {
+            "Type": "Jump 5 meters every step",
```
Comment (gioragutt) [inline server/src/database/BeastsDatabase.json]:
```suggestion
            "Type": "Jump 5 meters forward",
```

File: server/src/database/BeastsDatabase.json (hunk 1 above)
Comment (gioragutt) [inline server/src/database/BeastsDatabase.json]:
```suggestion
              "Type": "Jump 10 meters forward or over a wall"
```

**Pull request merged**